database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -18,7 +18,6 @@ public function up()
             $table->string('name');
             $table->string('email')->unique();
             $table->string('password');
-            $table->integer('access_level_id');
```

Comment:
Moet er niet background bij of doen we dat ergens anders storen, want waar word opgeslagen welke background van toepassing is bij welke user?

Reply:
Dat mag, maak er maar wat voor

Reply:
Okii